finetune/sequence_labeling.py

```diff
@@ -220,4 +231,31 @@ def _predict_op(self, logits, **kwargs):
         return label_idxs, label_probas
 
     def _predict_proba_op(self, logits, **kwargs):
-        return tf.no_op()
\ No newline at end of file
+        return tf.no_op()
+
+
+class SequenceMultiLabeler(SequenceLabeler):
```

Comment:
Odd class name but I don't have a better suggestion at the moment.

Reply:
I think we talked about making this a config argument rather than a class.  Looks like you've already done that for the most part.